README.md

```diff
@@ -30,7 +30,7 @@ $ mkdir $HOME/.local/share/signal-cli
 
 ```bash
 $ sudo docker run -d --name signal-api --restart=always -p 8080:8080 \
-      -v $HOME/.local/share/signal-cli:/home/.local/share/signal-cli \
+      -v $HOME/.local/share/signal-api:/home/.local/share/signal-cli \
       -e 'MODE=native' bbernhard/signal-cli-rest-api
 ```
```

Comment:
Are you sure that this worked for you out of the box? :thinking: 

I would assume that you would also need to adapt the line `mkdir $HOME/.local/share/signal-cli` above then. Otherwise you create a folder with the name `signal-cli` on your host system, but bind-mount the `signal-api` folder (which most probably doesn't exist).

Reply:
Yes, you're right.  I missed that bit, as I always create my container folders based on the container name, which is signal-api.  I've updated my pull request.